gswrap/__init__.py

```diff
@@ -45,25 +50,9 @@ def __init__(self, bucket: str, prefix: str) -> None:
         self.prefix = prefix
 
 
-class _Path:
+def classify(res_loc: str) -> Union[_GCSURL, str]:
```

Comment:
Sorry for not being very clear. I think this needs renaming again. How about `resource_type()`? Lets discuss your ideas offline